spec/acceptance/mod_php_spec.rb

```diff
@@ -4,6 +4,29 @@
 unless fact('operatingsystem') == 'SLES' && fact('operatingsystemmajrelease') == '12'
   describe 'apache::mod::php class' do
     context 'default php config' do
+      if ['16.04', '18.04'].include?(fact('operatingsystemmajrelease'))
+        # this policy defaults to 101. it prevents newly installed services from starting
+        # it is useful for containers, it prevents new processes during 'docker build'
+        # but we actually want to test the services and this should not behave like docker
+        # but like a normal operating system
+
+        # without this apache fails to start -> installation of mod-php-something fails because it reloads apache to enable the module. WTF Ubuntu
+        # exit codes are documented at https://askubuntu.com/a/365912. Default for docker images is 101
+        shell("sed -i 's/^exit.*/exit 0/' /usr/sbin/policy-rc.d")
+      end
+      if fact('operatingsystemmajrelease') == '18.04'
+        # Again Ubuntu. W T F.
```

Comment:
I'd just like to point out this is an upstream Debian bug, so your 'WTF Ubuntu' is misdirected and misplaced. Regardless your personal feelings and reactions should most likely never appear in public code in the first place even if misguided.

Reply:
sorry, that was written after hours of personal frustration, I removed it

Reply:
I know the feeling, thanks for cleaning it up!